Firestore/src/V1/Gapic/FirestoreGapicClient.php

```diff
@@ -770,14 +786,17 @@ public function listCollectionIds($parent, array $optionalArgs = [])
      *
      * @throws ApiException if the remote call fails
      */
-    public function listDocuments($parent, $collectionId, array $optionalArgs = [])
+    public function listDocuments($parent, array $optionalArgs = [])
```

Comment:
This will be fixed in https://github.com/googleapis/gapic-generator-php/commit/996dc498a726ae634be833f0ff0b4e59cacf6dd7